components/Card/Card.tsx

```diff
@@ -55,6 +85,48 @@ export const Card = (CardProps: CardProps) => {
       {cardItems?.length && cardItems?.length > 3 && (
         <SeeMore onClick={handleSeeMoreClick} />
       )}
+      {cardObjectsToDisplay?.length && (
+        <CardBootstrap.Body className="px-0">
+          <CardBootstrap.Text className={styles.body}>
+            <Container>
+              {cardObjectsToDisplay.map((row, idx) => (
+                <Row key={idx}>
+                  {row.map((col, idxx) => {
+                    return (
+                      <Col key={idxx} xs={4} md={4} lg={4}>
+                        <div className="p-5 mx-5">
+                          <LabeledIcon
+                            key={idxx}
+                            idImage={`https://malegislature.gov/Legislators/Profile/170/${col.id}.jpg`}
+                            mainText="Sponsor"
+                            subText={
+                              <a
+                                href={`https://malegislature.gov/Legislators/Profile/${col.id}`}
+                              >
+                                {col.Name}
+                              </a>
+                            }
+                          />
+                        </div>
+                      </Col>
+                    )
+                  })}
+                </Row>
+              ))}
+            </Container>
+          </CardBootstrap.Text>
+          {numOfSponsors && numOfSponsors > 3 && (
+            <div className="px-0" style={{ height: "50px" }}>
+              <hr className="px-0" />
+              <SeeMore
```

Comment:
I like the SeeMore improvements! We should move these changes into the existing seemore element. Can we rename typeOfItems to itemLabel and add it as a Card prop? That way we can pass in "Sponsors" from the BilSponsors component.